### ISSUE TITLE
Add Support for Secure Credential Transfer Between Applications and OpenLorenz

### ISSUE THREAD

**Issue by churay** [LLNL-Collaboration/uiuc2015#2]
The basic infrastructure for transferring credentials from a remotely running application to the running OpenLorenz instance needs to be established so that an application's users can be identified and notified when data is transferred.  This transferral of credentials can be facilitated by using [munge](https://github.com/dun/munge) on both the running application (to securely send credentials) and OpenLorenz (to securely receive credentials).  As a first step, Conduit should be extended so that it can securely communicate the running user's identity to OpenLorenz.

Once this task is complete, it should be possible for OpenLorenz to associate a remotely running instance of Conduit with a user.

**Comment by ccordi** [comment]
Munge is installed on both the OpenLorenz server and the Conduit server. They share a key file and can successfully unmunge each other's munged credentials. Munge can be run as the munge user but only with --syslog parameter.

**Comment by ccordi** [comment]
Here is a description of the issue running without --syslog:

Running without --syslog returns this errror: 
~$ sudo -H -u munge bash -c '/usr/sbin/munged'
munged: Error: Logfile is insecure: group-writable permissions set on "/var/log"

 This bug seems to be reported here: https://bugs.launchpad.net/ubuntu/+source/munge/+bug/1287624 .

default /var/log/ permissions : drwxrwxr-x 9 root  syslog 4096
Removing the syslog write permissions from /var/log/ does allow the service to start without issue, but forcing it to just allow syslog write access with --syslog seems like a better idea.

**Comment by churay** [comment]
The only hang up at this point seems to be using "mrsh," which is only a problem because of the NAT settings on the AWS clusters.  I'll move this problem to a separate issue and close this for now.
